app/javascript/pages/MyEvents/index.js

```diff
@@ -505,6 +505,7 @@ const individualEventInputToOptimisticResponse = (data, input) => {
     date: moment(input.date),
     duration: input.duration,
     eventType: R.find(et => et.id === input.eventTypeId, data.eventTypes),
+    tags: R.filter(tag => R.find(inputTag => inputTag.id === tag.id, input.tags), data.tags),
```

Comment:
can also use `R.eqProps` the advantage here is it won't crash if `id` doesn't exist.
```
tags: R.filter(tag => R.includes(R.eqProps('id', tag), listB), listA)
```